### PR TITLE
Link tutorial to workflow

### LINKED ISSUE #1784: allow linked tutorial resource for workflow update_links
`https://panoptes-staging.zooniverse.org/api/workflows/1817/links/tutorials` returns 404 since it's not an allowed route [see here](https://github.com/zooniverse/Panoptes/blob/master/config/routes.rb#L95).

- [x] specify the route links, add the linking permissions from the above link
- [x] as well as the workflow update_schema [here](https://github.com/zooniverse/Panoptes/blob/9ed465843998358a9bfdaa10d3779afb58b7036a/app/schemas/workflow_update_schema.rb#L73)
- [ ] audit the workflow links and remove unused ones (seems the :aggregation & subject_queue resource links aren't used)

### PR DESCRIPTION
**Pull request by zwolf**
Another WIP PR to fix #1784. Updates to routing and JSON validation were simple, but I'm not sure exactly how the spec should look. The WorkflowsController and Updatable spec files shed a little light, but I'd appreciate a few extra eyes.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: spec/controllers/api/v1/workflows_controller_spec.rb [hunk 1]
```diff
@@ -358,6 +358,38 @@
...
 
+    context 'linking a tutorial' do
+
+      let(:tutorial_project) { project }
+      let(:linked_resource) { create(:tutorial, project: tutorial_project) }
+      let(:resource) { create(:workflow) }
+      let(:resource_id) { :workflow_id }
+      let(:test_relation) { :tutorials }
+      let(:test_relation_ids) { [ linked_resource.id.to_s ] }
+      # let(:update_params) { { workflow: resource.id.to_s, tutorials: test_relation_ids, link_relation: "tutorials" }}
```
Comment (marten) [inline spec/controllers/api/v1/workflows_controller_spec.rb]: Mind removing these commented-out lines?

**Comment by marten** [comment]
LGTM

**Comment by zwolf** [comment]
Closes #1784

File: spec/controllers/api/v1/workflows_controller_spec.rb [hunk 2]
```diff
@@ -66,8 +66,9 @@
     end
   end
 
-  describe '#update' do
+  describe '#update', :focus do
```
Comment (marten) [inline spec/controllers/api/v1/workflows_controller_spec.rb]: focus here

**Pull request merged**